src/modules/hip/kernel/down_mixing.hpp

```diff
@@ -0,0 +1,64 @@
+#include <hip/hip_runtime.h>
+#include "rpp_hip_common.hpp"
+
+__global__ void down_mixing_hip_tensor(float *srcPtr,
+                                       uint srcStride,
+                                       float *dstPtr,
+                                       uint dstStride,
+                                       int *srcDimsTensor)
+
+{
+    int id_x = hipBlockIdx_x * hipBlockDim_x + hipThreadIdx_x;
```

Comment:
Why is this not vectorizeed?

Reply:
we can only vectorize the channel code as how it is done in host code
Initially felt it might not be efficient to vectorize the channel loop in HIP
But explored further and vectorized the code now and checked the performance. It looks fine